src/styles/typography.stories.scss

```diff
@@ -0,0 +1,61 @@
+@import './base';
+@import 'variables';
+
+.typography {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    margin-top: 25px;
+
+    .weights {
+        display: flex;
+        flex-wrap: wrap;
+        margin-top: 20px;
+        color: $black;
+    }
+
+    .weights .preview {
+        font-size: 72px;
+        line-height: 72px;
+    }
+
+    .weights .regular {
+        margin-right: 15px;
+    }
+
+    .weights .bold .preview {
+        font-weight: bold;
+        font-family: 'Lato', sans-serif;
+    }
+
+    .font-container,
+    .headings-container {
+        width: 33%;
+    }
+
+    .heading {
+        display: flex;
+        align-items: center;
+        line-height: 32px;
+    }
+
+    label {
+        margin-right: 10px;
+        color: $great-balls-of-fire;
```

Comment:
not an exact swap but we probably replace with `$bdl-watermelon-red` or `$bdl-orange` since this color is deprecated